presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -843,6 +844,15 @@ protected Void visitDropTable(DropTable node, Integer context)
         }
 
         @Override
+        protected Void visitTruncateTable(TruncateTable node, Integer context)
+        {
+            builder.append("TRUNCATE TABLE ");
+            builder.append(node.getTableName());
```

Comment:
This should be:

```
.append(formatName(node.getTableName()));
```

Otherwise, the output will be invalid if the table contains illegal sql identifier characters. `formatName` will quote identifiers if necessary.